### DISCUSSION TITLE
Card Color Customization

### DISCUSSION THREAD

**Discussion by untillnesss** [filamentphp/filament#8398] [Help] [answered]
How can I change the background color of this Card Component?

![image](https://github.com/filamentphp/filament/assets/33585758/d99a52db-ee14-4e37-8e40-f9d7427497f5)

I've added the existing Class in Tailwind, but it still doesn't change.
There is also no documentation yet.

**Comment by leandrocfe** [comment, marked as answer]
You can [create a custom theme](https://filamentphp.com/docs/3.x/panels/themes#creating-a-custom-theme)

```html
<x-filament::section class="custom-section">
    ...
</x-filament::section>
```
```css
/*resources/css/filament/admin/theme.css*/
.custom-section {
    @apply bg-green-800;
}
```
